src/main/resources/application.yml

```
@@ -66,6 +66,22 @@ spring:
       ddl-auto: create-drop
     show-sql: true
 
+---
+spring:
+  config:
+    activate:
+      on-profile: docker
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)